crates/dynamic-config/src/manager.rs

```diff
@@ -0,0 +1,72 @@
+use anyhow::{anyhow, Result};
+
+use gw_config::{Config, FeeConfig};
+use gw_tx_filter::{
+    erc20_creator_allowlist::SUDTProxyAccountAllowlist,
+    polyjuice_contract_creator_allowlist::PolyjuiceContractCreatorAllowList,
+};
+
+use crate::{fee_config::FeeConfigManager, whitelist_config::WhilteListConfigManager};
+
+// Some configs can be hot reloaded through DynamicConfigManager.
+// So that we don't need to restart to take effect every time.
+#[derive(Default)]
+pub struct DynamicConfigManager {
+    config_github_url: Option<String>,
+    fee_manager: FeeConfigManager,
+    whitelist_manager: WhilteListConfigManager,
+}
+
+impl DynamicConfigManager {
+    pub fn create(config: Config) -> Self {
```

Comment:
We should create new structure DynamicConfig or `RemoteConfig` to only include the configurations that can can be dynamically reloaded.

Use Config structure introcing too much complexity.